docs/_adapters/nginx-sm.md

```diff
@@ -12,8 +12,7 @@ permalink: service-meshes/adapters/nginx-sm
 ---
 {% include adapter-status.html %}
 
-The {{ page.name }} is currently under construction ({{ page.project_status }} state). Want to contribute? Check our [progress](page.github_link).
-
+The {{ page.name }} is currently under construction ({{ page.project_status }} state). Want to contribute? Check our [progress](https://github.com/layer5io/meshery-nginx-sm).
```

Comment:
Actually, the link is correct but the way it was rendered was incorrect. Check the code at the bottom of this file, where the same link is used and update here accordingly.